src/leds.c

```diff
@@ -322,6 +323,32 @@ static void anim_felony(Leds *leds, const LedStrip *strip, const LedBar *bar, fl
     }
 }
 
+static void anim_rainbow_cycle(Leds *leds, const LedStrip *strip, float time) {
+    uint8_t count = 10;
+    uint8_t segment = 256 / count;
```

Comment:
I'd consider making the `segment` float so that there's no bigger segment at the end due to the rounding. Also, use `255` instead of 256.

And can you make both the above variables `const`? Higher chances for the compiler to optimize away the division.